schemas/entry.schema.json

```diff
@@ -80,11 +80,68 @@
       "additionalProperties": false,
       "properties": {
         "breakage": {
-          "description": "List of URLs to issues where actual site breakage is tracked. If available, point the link directly to a comment with clear STR",
+          "description": "List of issues tracking known site breakage.",
           "type": "array",
           "items": {
-            "type": "string",
-            "format": "uri"
+            "anyOf": [
+              {
+              "type": "string",
+              "format": "uri",
+              "description": "URL of tracking issue"
+            },
+              {
+              "type": "object",
+              "properties": {
+                "url": {
+                  "type": "string",
+                  "format": "uri",
+                  "description": "URL of tracking issue"
+                },
+                "site": {
+                  "type": "string",
+                  "format": "uri",
+                  "description": "URL of broken site of page"
+                },
+                "platform": {
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "enum": ["all", "desktop", "mobile", "windows", "macos", "linux"],
+                    "description": "List of affected platforms. Default is 'all'"
+                  }
+                },
+                "last_reproduced": {
+                  "type": "string",
+                  "format": "date",
+                  "description": "Most recent date the issue was successfully reproduced"
+                },
+                "intervention": {
+                  "type": "string",
+                  "format": "uri",
+                  "description": "URL of intervention that is shipping or has been shipped. Link to the code in the GitHub repository, and use the canonical URLs to ensure persistance over time"
+                },
+                "impact": {
+                  "type": "string",
+                  "enum": ["site_broken", "feature_broken", "significant_visual", "minor_visual", "unsupported_message"],
+                  "description": "Type of breakage"
+                },
+                "affects_users": {
+                  "type": "string",
+                  "enum": ["all", "some", "few"],
+                  "description": "What fraction of users are affected. 'all' where any site user is likely to run into the issue, 'some' for issues that are common but many users will not experience, and 'few' where the breakage depends on an unusual configuration or similar."
+                },
+                "resolution": {
+                  "type": "string",
+                  "enum": ["site_changed", "site_fixed"],
```

Comment:
This raises the question again of "what do we do with KB-entries about platform issues that have been fixed".

If we, in some future, fix `<marquee>`. Do we delete `marquee.yml`? Do we remove all site breakage because we no longer see some? Do we move the entry into a "fixed" folder? imho, having historic data is good, because it allows us to go back in time and check back on things we've succeeded on.

Keeping that in mind, we probably need different "fixed" resolutions, depending on what happened:

- `fixed_by_intervention`
- `fixed_by_platform_change`
- `fixed_by_site`

or something along those lines.

Reply:
I think if the platform issue is fixed we should either a) have some top-level status field that records that or b) move the entry into a different directory. I think I favour b) because it seems easier to implement and also probably makes tooling easier (you just ignore all the "fixed" issues for most operations by not looking in that directory).

My thinking for not considering "fixed_by_intervention" as a "fixed" status is that the issue isn't really fixed and we already have a field to link to the intervention, if there is one.

Reply:
> that the issue isn't really fixed and we already have a field to link to the intervention, if there is one.

This is true, but that distinction is relevant for our ranking. If we have an issue with critical breakage on critical sites, but we can and do ship interventions for that, the issue should probably be ranked lower than an issue we can't ship interventions for.

Reply:
Right, but I was imagining the presence of an intervention would cut the computed priority quite a lot. Are there cases where we have an intervention but the issue isn't really "fixed" by the intervention?

Reply:
I don't think so. If we can't completely fix an issue with an intervention, we usually don't even bother shipping it. So if you want to take the presence of an intervention as an additional signal, not just the state, then yeah, your version works fine. :)